pages/windows/rdpsign.md

```diff
@@ -0,0 +1,23 @@
+# rdpsign
+
+> A tool for signing Remote Desktop Protocol (RDP) files.
+
+- Sign an RDP file:
+
+`rdpsign {{path/to/file.rdp}}`
+
+- Sign an RDP file using a specific sha256 hash:
+
+`rdpsign {{path/to/file.rdp}} /sha265 {{hash}}`
+
+- Enable quiet output:
+
+`rdpsign {{path/to/file.rdp}} /q`
+
+- Display verbose warnings, messages and statuses:
+
+`rdpsign {{path/to/file.rdp}} /v`
+
+- Test the signing and output the result without updating the input:
```

Comment:
Maybe just say - "Test the signing by displaying the output to console without updating the file" ?

Reply:
Just interested, is it better to say `output to console` or `output to stdout`?

Reply:
I'd probably go with `stdout` myself, as it makes sense in both interactive and non-interactive contexts.

Reply:
I'm fine with both. I think in other pages, we have a mixture of both console and stdout.

Reply:
Went with `stdout` :+1: